system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -58,6 +58,8 @@
 
 #define PWD_NEVER_EXPIRES (__int64)0x8000000000000000
 
+#define UNK_PERM_VALUE (SecAccessFlags)-2	//used to initialize "default" permission, which we later try to deduce
```

Comment:
Yes it likely could, but a scan of the code shows there are a lot of places where it checks for things like "if (perms < SecAccess_None)" and then takes some action. I prefer to make this PR as benign as possible (day before the release!) and will address with https://track.hpccsystems.com/browse/HPCC-16337